### ISSUE TITLE
Colon symbol is not treated as a word separator

### ISSUE THREAD

**Issue by polariton** [software-jessies-org/jessies#29]
Hello,

The Terminator does not treat the colon symbol ":" as a word separator. The most inconvenient consequence of this mistake is a wrong selection of words via double-click.

How to reproduce? Let's assume we have the command line prompt:
user@host:~/.config/terminator $
which is obtained by the following command: export PS1="\u@\h:\w $ ".

When I double-click on the ".config/terminator", the Terminator incorrectly selects both the path and hostname: "host:\~/.config/terminator". The Gnome terminal and many others correctly select only the path "~/.config/terminator".

Best regards,
Stan

**Comment by enh** [comment]
yeah, if i remember rightly IPv6 addresses were the reason behind this choice. but i have to admit that i've found it more annoying than helpful. the trickiest case is something like `EventQueue.java:721` --- whether or not you want the colon there really depends on context.

despite the fact that other terminal emulators (like gnome terminal) have _removed_ this option, it does seem like there's a reasonable argument for the selection stop characters being configurable.

**Comment by altaurog** [comment]
For me, treating `:` as a word character is usually the desired behavior.  Consider urls, for example.
